source/sound/SoundMgr.cpp

```diff
@@ -4,6 +4,7 @@
 #include "SoundBase.h"
 #include "SoundBaseMgr.h"
 #include <OgreDataStream.h>
+#include <OgreException.h>
```

Comment:
This doesn't seem needed right now to compile with Ogre 1.9, but it will fail later on with Ogre 1.10+. Since we're using `Exception` in this file, best be explicit anyway.